src/main/java/org/example/homework6/BasicInteractiveCalculator.java

```diff
@@ -0,0 +1,44 @@
+package org.example.homework6;
+
+import java.util.Scanner;
+
+public class BasicInteractiveCalculator {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter a number");
+        double firstNumber = Double.parseDouble(scanner.nextLine());
+
+        System.out.println("Enter an operation (+, -, *, /)");
+        char arithmeticSymbol = scanner.nextLine().charAt(0);
+
+        System.out.println("Enter another number");
+        double secondNumber = Double.parseDouble(scanner.nextLine());
+
+        if (arithmeticSymbol == '/' && secondNumber == 0){
```

Comment:
It is not a good idea to handle this case out of your switch `case '/' ->` condition